listing-bundle/src/Resources/contao/modules/ModuleListing.php

```diff
@@ -141,7 +141,7 @@ protected function compile()
 		}
 
 		$strQuery .= $strWhere;
-		$objTotal = $this->Database->prepare($strQuery)->execute($varKeyword);
+		$objTotal = $this->Database->prepare($strQuery)->execute(...($varKeyword ? array($varKeyword) : array()));
```

Comment:
This looks strange. Isn't `$varKeyword` always a string?

Reply:
Yes, but could be `''` if the placeholder does not exist.

Reply:
Maybe `$varKeyword` should be `array()` in line 102 and `array('%' . $strFor . '%')` in line 123? Then we could use just `...$varKeyword` here.

Reply:
done in a31ec9ed1ebf4e5e4e688684688665a9a40214b2